README.md

```diff
@@ -64,13 +65,16 @@ client.trackTrace("trace message");
 ### Telemetry Processor
 
 ```javascript
-public addTelemetryProcessor(telemetryProcessor: (envelope: ContractsModule.Contracts.Envelope) => boolean)
+public addTelemetryProcessor(telemetryProcessor: (envelope: ContractsModule.Contracts.Envelope, context: { http.RequestOptions, http.ClientRequest, http.ClientResponse, userProperties }) => boolean)

```

Comment:
The original context variable for telemetry processors was added by @SergeyKanzhelev in a recent PR. I think it might be cleaner to not pass the correlation context as userProperties if automatic correlation is enabled, but rather as the entire object (removing these http.Request related fields and leaving it to the user to add these to the correlation context if they're interested in them). These http related fields could be provided as the context if automatic correlation is disabled.

Thoughts?